addons/website_crm_partner_assign/models/res_partner.py

```diff
@@ -38,6 +39,18 @@ class ResPartnerActivation(models.Model):
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
+    @api.model
+    def default_get(self, fields_list):
+        default_vals = super().default_get(fields_list)
+        if self.env.context.get('set_default_grade_activation'):
```

Comment:
Maybe ``partner_set_default_grade_activation`` to prefix with model ?

Reply:
Done